homeassistant/components/bluesound/media_player.py

```diff
@@ -203,33 +203,23 @@ async def async_service_handler(service):
 class BluesoundPlayer(MediaPlayerEntity):
     """Representation of a Bluesound Player."""
 
-    def __init__(self, hass, host, port=None, name=None, init_callback=None):
+    _attr_media_content_type = MEDIA_TYPE_MUSIC
+
+    def __init__(self, hass, host, port=DEFAULT_PORT, name=None, init_callback=None):
         """Initialize the media player."""
         self.host = host
         self._hass = hass
         self.port = port
         self._polling_session = async_get_clientsession(hass)
         self._polling_task = None  # The actual polling task.
-        self._name = name
-        self._icon = None
-        self._capture_items = []
-        self._services_items = []
-        self._preset_items = []
+        self._attr_name = name
+        self._capture_items = self._services_items = self._preset_items = []
```

Comment:
This makes all three class variables point to the same list.

Reply:
Hm. That does change things.